client/app/components/sections/SearchPage/SearchPage.js

```diff
@@ -70,21 +72,33 @@ class SearchPage extends Component {
   render() {
     const { marketplace_color1: marketplaceColor1, displayBrandingInfo, linkToSharetribe } = { ...DEFAULT_CONTEXT, ...this.props.marketplace };
     const displayBranding = this.props.marketplace && displayBrandingInfo && linkToSharetribe;
-    return div({ className: css.searchPage }, [
+    return div({
+      className: classNames('SearchPage', css.SearchPage),
+    }, [
       r(Topbar, {
         ...this.props.topbar,
         routes: this.props.routes,
       }),
-      r(ListingCardPanel,
-        { className: css.listingContainer },
-        this.listings.map((listing) =>
-          r(ListingCard, this.listingProps(listing, marketplaceColor1))
-      )),
-      div({}, [
-        `${this.props.searchPage.state.get('total')} listings, page ${this.props.searchPage.state.get('page')}/${this.totalPages}`,
-        this.hasPrevPage ? a({ onClick: this.setPage(this.prevPage()), href: this.pageUrl(this.prevPage()) }, 'prev') : null,
-        this.hasNextPage ? a({ onClick: this.setPage(this.nextPage()), href: this.pageUrl(this.nextPage()) }, 'next') : null,
-      ]),
+      this.listings.size > 0 ?
```

Comment:
This branching is a bit difficult to see from the code, maybe put the searchpage with results and empty state to variables beforehand? E.g. `this.listings.size > 0 ? listingCards : emptyState`